ddtrace.gemspec

```diff
@@ -53,13 +53,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pimpmychangelog', '>= 0.1.2'
   spec.add_development_dependency 'appraisal', '~> 2.2'
   spec.add_development_dependency 'yard', '~> 0.9'
-  spec.add_development_dependency 'webmock', '~> 2.0'
+  spec.add_development_dependency 'webmock', '>= 3.10.0'
```

Comment:
We need webmock >= 3 for Ruby support. Thankfully it works correctly with all our supported Rubies.